### ISSUE TITLE
GetAsNullableString Doesn't work as expected, Get() function can't return nil

### ISSUE THREAD

**Issue by antonio-alexander** [pip-services3-go/pip-services3-commons-go#4]
https://github.com/pip-services3-go/pip-services3-commons-go/blob/49fcb82efc82c8c84653c1036f6d49f76056300e/data/StringValueMap.go#L220

As coded it's impossible for this to return a nullable string when not present. The Get() function as coded won't panic when an item isn't present, but will return the default value (which is empty, but can never be nil). A solution would be code that did the following:

```go
func (c *StringValueMap) GetAsNullableString(key string) *string {
	value, ok := c.value[key]
	if !ok {
		return nil
	}
	return convert.StringConverter.ToNullableString(value)
}
```
I expect that a similar issue/bug is present for other nullable functions that use the Get() function. Maybe Contains() could be a solve for this problem?

**Comment by levichevdmitry** [comment]
Hello, @antonio-alexander!
Thank you for issue report! 
I was already fix this issue, please see new release [v1.1.5](https://github.com/pip-services3-go/pip-services3-commons-go/releases/tag/v1.1.5)

**Comment by antonio-alexander** [comment]
@levichevdmitry thanks for your quick response!
